tests/pgsql/pgsql-bug-6983-ids/test.yaml

```diff
@@ -0,0 +1,25 @@
+requires:
+  min-version: 7.0
+
+pcap: ../pgsql-ssl-rejected-md5-auth-simple-query/input.pcap
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 7
+    match:
+      event_type: pgsql
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 1
```

Comment:
@jufajardini was it not the test for pgsql metadata in alerts ? (apparently not)